modules/nf-core/narfmap/align/tests/main.nf.test

```diff
@@ -129,10 +133,12 @@ nextflow_process {
             assertAll (
                 { assert process.success },
                 { assert snapshot(
+                    process.out.bam,
+                    process.out.versions,
                     file(process.out.bam[0][1]).name,
```

Comment:
I think you can remove this line as well,since you're snapshotting process.out.bam already

Reply:
I just copied what was at bwa/mem 🤷🏻‍♂️ 

https://github.com/nf-core/modules/blob/58ebf878bd5285a3de23cd52f14a343b7cd47d98/modules/nf-core/bwa/mem/tests/main.nf.test#L45-L53

Reply:
I see

Reply:
but in your case, since you snapshot `process.out.bam`, you don't need to snapshot `file(process.out.bam[0][1]).name` as well

Reply:
I think we need to update the bwa/mem tests with the nft-bam plugins

Reply:
Made an issue for updating those and cleaned this one up for now.